README.rst

```diff
@@ -56,66 +56,117 @@ Within your python file or notebook:
 .. code-block:: python
 
     from datenguidepy import Query
+    from datenguidepy import get_all_regions
+    from datenguidepy import get_statistics
 
 **2. Creating a query**
 
-- either for single regions
+Start with querying either for single regions:
 
 .. code-block:: python
 
-    query = Query.region('01')
+    q = Query.region('01')
 
-- or for all subregions a region (e.g. all Kommunen in a Bundesland)
+or for all subregions within a region (e.g. all Kommunen in a Bundesland)
 
 .. code-block:: python
 
    query_allregions = Query.all_regions(parent='01')
 
-- How to get IDs for regions? see below "Get information on fields and meta data"
+- How to get IDs for regions?
+
+.. code-block:: python
+
+    # Overview of region IDs
+    get_all_regions()
+
+Use pandas *query()* functionality to filter according to level, e.g. for Bundesländer *"nuts1"*
+
+.. code-block:: python
+
+    # Filtered for Bundesländer (federal states)
+    get_all_regions().query("level == 'nuts1'")
+
+See below "Get information on fields and meta data" for more options on regions.
 
 **3. Add statistics (fields)**
 
-- Add statistics you want to get data on
+Add statistics to your query for which you want to get data
+
+.. code-block:: python
+
+    stats = q.add_field('BEV001')
+
+- How do I find the short name of the statistics?
+
+.. code-block:: python
```

Comment:
Does this need to be a code block?

Reply:
Not necessarily, I wanted to differentiate it from the main text, that's all. I suppose this can also be updated, once there is a quicker way to access the short names.